src/openrct2-ui/libopenrct2ui.vcxproj

```diff
@@ -53,4 +213,4 @@
     </Lib>
   </ItemDefinitionGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
-</Project>
+</Project>
```

Comment:
*cough*

Reply:
The problem is vs is going to do this every time it edits the file.

Reply:
I'd rather leave this change and prevent us from opening a new issue every time someone edits those files on windows.

Reply:
The counter argument is that anyone not on Windows will insert the end of line when they edit the file.

Reply:
Will they, though? Because I remember there was one file which I kept doing this and reverting and when we settled on "letting it slip" we didn't have this problem anymore, as far as I can remember